crates/types/src/data.rs

```diff
@@ -740,13 +740,13 @@ pub mod null_block {
                 [0_u8; 32], 0,
             );
 
-        let (null_block, null_block_metadata) =
+        let (_null_block, null_block_metadata) =
```

Comment:
Don't need to fix this now, but ignoring the block here is a smell and I think the specific problem is that `commitment` should take the payload, rather than assuming that the byte encoding of an empty payload is empty bytes (it is for now, though, so this isn't urgent)